app/components/welcome/welcome.js

```diff
@@ -82,8 +82,8 @@ const dialogs = [
       <Heading>As-you-go, not after-the-fact</Heading>
       <p>
         You are probably used to doing research, writing a full paper, and then
-        going through the laborious process of finding a journal. With
-        Hypergraph, you can publish each step of your research as-you-go.
+        going through the laborious process of becoming accepted by a journal.
```

Comment:
```suggestion
        going through the laborious process of getting accepted by a journal.
```

Becoming accepted sounds very existential, as researcher I'd use this phrasing (publication gets accepted).